### ISSUE TITLE
Feedback - Quizzes with 'Awaiting Teacher Grade' has a 'Pending teacher grade' button that doesn't open

### ISSUE THREAD

**Issue by kspilarski** [WordPress/Learn#2686]
## Type of feedback
//content

## Description
In quizzes awaiting teacher grade like https://learn.wordpress.org/quiz/mentoring-community-organizers-3/ after taking the quiz, it will display a "Pending teacher grade" button that is clickable but doesn't open anywhere. 

Is  "Pending teacher grade" supposed to be a button or just a label? 

## Step-by-step reproduction instructions (optional)
1. Go to https://learn.wordpress.org/quiz/mentoring-community-organizers-3/ or in lessons with quizzes that need to be graded by a teacher. 
2. Complete the quiz.
3. Click the "Pending teacher grade" button that appears. 

## Screenshots or screen recording (optional)
![Screen Capture on 2024-07-16 at 18-53-08](https://github.com/user-attachments/assets/be355505-6524-4295-bc7b-08d19bc512f8)

## Other information (optional)
Chrome 124, macOS 14.5

**Comment by kathrynwp** [comment]
> Is "Pending teacher grade" supposed to be a button or just a label?

I think it's just a message – rather than a button – since as you say, the learner is waiting for a teacher to submit a grade. You're right, it's confusing that this looks like a button, and should be redesigned to not look like a button.

Flagging for @adamwoodnz to confirm.

**Comment by adamwoodnz** [comment]
Agree, this should not be a button, thanks for raising!

**Comment by outdoor2kode** [comment]
@fcoveram could you update the design for this?

Currently it looks like this, but it shouldn't be a button since it's just meant to inform.

![image](https://github.com/user-attachments/assets/e66e9edc-5d9c-4871-9704-eb3dc094e9dc)

**Comment by outdoor2kode** [comment]
Also notice that when completing the quiz and waiting for the results, if you return to the lesson associated with the quiz, there's a notice displayed.

![image](https://github.com/user-attachments/assets/45ce176e-9b7c-419b-8929-1127d1802809)

Its styles are broken and need to be updated as well. @fcoveram

**Comment by fcoveram** [comment]
I hope to have understood the flow correctly. Here are the designs for the whole flow.

## Quiz completed with waiting message

![Mockup of quiz completed with a waiting message](https://github.com/user-attachments/assets/7778ef29-f1f5-44dd-880b-a498f3c3a70c)

When users complete the quiz, the page refreshes and shows a message saying that answers were submitted and the quiz will be graded soon.

In this page, I replaced the default "Continue to next lesson" but with a "Back to lesson." This action prevents users to continue to the next one without knowing the pending state. Mainly because the design of success message might hide the awaiting state part. What do you think of this idea @kathrynwp?

## Lesson with waiting message

![Mockup of lesson with waiting message](https://github.com/user-attachments/assets/f95630cf-770e-4595-9d53-dcd67f7fa17e)

On this page, users can go to the previous and next lesson, but the waiting message is placed above the browsing actions. The message copy could be improved in conveying that the lesson awaits a grade.

## Quiz. Viewing quiz responded

![image](https://github.com/user-attachments/assets/0621d4be-6044-4567-b080-19fc3d9f7392)

If the user clicks on "View quiz", the page shows an info callout saying that this quiz awaits for a grade. Now, the CTA allows users to continue to next lesson.

---

How does the above sound?

I'm curious about what happens once the teacher assigns a grade, and when users restart the quiz after responding it. And also, could you help us @thetinyl suggesting copy for each message pointed out above?

**Comment by kathrynwp** [comment]
@renintw I wonder, would it be possible to share a little screencast of the flow to help both @fcoveram and I better visualize the steps?

I also just also wanted to mention that the Community team is going to be modifying all courses that currently have open-ended questions which require grading, since no one has actually been grading these – meaning that folks are waiting indefinitely. That doesn't mean we may never have open-ended questions again in the future, though, so I still think it's worth fixing this flow.
